src/modules/home/routes/single-property.route.tsx

```diff
@@ -31,6 +39,21 @@ export function SinglePropertyRoute() {
 
   const accommodationData: Accommodation = data.data;
 
+  const handleReserve = () => {
+    if (!checkInDate || !checkOutDate) {
+      toast.error('Please select check-in and check-out dates.');
+      return;
+    }
+
+    createBooking.mutate({
+      checkInDate: checkInDate.toDate(),
+      checkOutDate: checkOutDate.toDate(),
+      accommodationId: id,
+      totalPrice: accommodationData.pricePerNight,
+      numberOfGuests: 1,
```

Comment:
is it the default value? are users allowed to choose the number of guests when making a booking?